### ISSUE TITLE
Not working on 2a

### ISSUE THREAD

**Issue by shadowe1ite** [SyedAhkam/glyph-studio#8]
I tried recoding and taping on the led buttons but glyphs interface not lighting

**Comment by sedbytes** [comment]
Hi, did you turn on glyph developer mode?

**Comment by shadowe1ite** [comment]
> Hi, did you turn on glyph developer mode?

Yes, also glyph debugging

**Comment by sedbytes** [comment]
Do you have a developer background?

I would need someone to help me with error logs captured using `adb logcat` to debug this further as I don't own a 2a.

**Comment by shadowe1ite** [comment]
> Do you have a developer background?
> 
> I would need someone to help me with error logs captured using `adb logcat` to debug this further as I don't own a 2a.

Yeah I can do that. I'll share that within 3 days. I don't have access to a PC right now, but I will soon."

**Comment by shadowe1ite** [comment]
Tell me if you need any other logs

**Comment by sedbytes** [comment]
@shadowe1ite update?
